src/assets/scripts/client/App.js

```diff
@@ -50,8 +51,55 @@ export default class App {
         this.prop.log = LOG.DEBUG;
         this.prop.loaded = false;
 
-        return this.setupHandlers()
-            .loadInitialAirport(airportLoadList, initialAirportToLoad);
+        $.getJSON('assets/airports/airportLoadList.json')
+            .done((data) => {
```

Comment:
this callback should also live within a bound handler.  something like `onSuccessHandler` or similar

Reply:
`onAirportLoadListFetchedHandler` bound to `_onAirportLoadListFetched`